tsconfig.json

```
@@ -16,7 +16,19 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
```

Se trocar por

```suggestion
    "baseUrl": "./src",
```

acho que não precisa incluir manualmente subdiretórios na chave "paths" abaixo